src/interface_adapter/login/LoginPresenter.java

```diff
@@ -5,8 +5,10 @@
 import interface_adapter.logged_in.LoggedInViewModel;
 import use_case.login.LoginOutputBoundary;
 import use_case.login.LoginOutputData;
+import use_case.switch_view.SwitchViewOutputBoundary;
+import use_case.switch_view.SwitchViewOutputData;
 
-public class LoginPresenter implements LoginOutputBoundary {
+public class LoginPresenter implements LoginOutputBoundary, SwitchViewOutputBoundary {
```

Comment:
We definitely want a SwitchViewPresenter

Reply:
Let's add this as a second PR